### PR TITLE
Sort references by key name

### PR DESCRIPTION
**Pull request by cenx1**
64bit declarations for AddIn

Add UTF8 file handling using binary read/write

Change FileSystemObject to ADODB.Stream for better UTF-8 compatability

Fix incorrect procedure call in RunAfterBuild

### CODE REVIEW

File: Version Control.accda.src/modules/clsDbVbeReference.bas [hunk 1]
```diff
@@ -56,6 +56,9 @@ Private Sub IDbComponent_Export()
         dItems.Add ref.Name, dRef
     Next cRef
     
+    ' Sort dictionary by keys, so the order is the same in exported file
+    Set dItems = SortDictionaryByKey(dItems)
```
Comment (joyfullservice) [inline Version Control.accda.src/modules/clsDbVbeReference.bas:60]: Is this primarily for having a more organized list? I like that in concept, but references are evaluated by the order in which they are saved in the database, which would be important to preserve. I will probably roll back this change in the master, but feel free to keep the dialog going.

Comment (cenx1) [reply Version Control.accda.src/modules/clsDbVbeReference.bas:60]: Well, I implemented the sorting due to me and my coworker kept getting the references in different orders after export and build and that shows up in the git changed files window (without any acutal changes except for the order).

Comment (cenx1) [reply Version Control.accda.src/modules/clsDbVbeReference.bas:60]: Yes, it is. The references come out in random order after build and export, hence it shows up in the git changed files window without any actual changes (except for the order)

Comment (joyfullservice) [reply Version Control.accda.src/modules/clsDbVbeReference.bas:60]: Interesting... I wonder if we need to do something to enforce the order. I was only managing the non-built-in references in the json file. Was the order changing for those? I would expect that they would be in the same order that they were added...

**Pull request merged**